Formula/graphite2.rb

```diff
@@ -28,7 +41,7 @@ def install
   test do
     resource("testfont").stage do
       shape = shell_output("#{bin}/gr2fonttest Simple-Graphite-Font.ttf 'abcde'")
-      assert_match /67.*36.*37.*38.*71/m, shape
+      assert_match(/67.*36.*37.*38.*71/m, shape)
```

Comment:
Oh boy ... that is an unfortunate warning given how many of these we have without parentheses.